File: java/estruturas-repetitivas/ParImpar.java
import java.util.Scanner;

public class ParImpar {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num, x, i;

        System.out.print("Quantos numeros voce vai digitar? ");
        x = sc.nextInt();

        for (i = 0; i < x; i++) {
            System.out.print("Digite um numero: ");
            num = sc.nextInt();

            if (num == 0) {
                System.out.println("NULO");
            } else {
                if (num % 2 == 0 && num < 0) {
                    System.out.println("PAR NEGATIVO");
                } else if (num % 2 == 0 && num > 0) {
                    System.out.println("PAR POSITIVO");
                } else if (num % 2 == 1 && num < 0) {
                    System.out.println("IMPAR NEGATIVO");
                } else {
                    System.out.println("IMPAR POSITIVO");
                }
            }
        }

        sc.close();
    }
}

File: js/estruturas-repetitivas/parImpar.js
const readline = require('readline-sync');

const X = Number(readline.question("Quantos numeros voce vai digitar? "));

for (let i = 0; i < X; i++) {
    let num = Number(readline.question("Digite um numero: "));
    if (num === 0) {
        console.log("NULO");
    } else {
        if (num < 0 && num % 2 !== 0) {
           console.log("IMPAR NEGATIVO"); 
        } 
        if (num >  0 && num % 2 !== 0) {
            console.log("IMPAR POSITIVO"); 
        }

        if (num < 0 && num % 2 === 0) {
            console.log("PAR NEGATIVO"); 
        }

        if (num > 0 && num % 2 === 0) {
            console.log("PAR POSITIVO"); 
        }
    }
}